test/web-platform-tests/to-upstream/css/selectors/dir-selector-auto-dont-upstream.html

```diff
@@ -0,0 +1,95 @@
+<!doctype html>
+<html>
+<head>
+  <link rel="help" href="http://www.w3.org/TR/selectors4/#dir-pseudo">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <!-- jsdom only test without innerText -->
```

Comment:
For this test you have a helpful comment explaining why we have a dont-upstream version.

Can you do the same for others?

Also maybe include the comment somewhere very high up like after `<!doctype html>`. That makes it easier to spot.